tests/importable_test.py

```diff
@@ -19,8 +20,31 @@ def test_entry_points_exposed(entry_points_group: str) -> None:
     pre-installed.
     """
     entry_points = _discover_entry_points(group=entry_points_group)
-    assert 'x' in entry_points.names
+    expected_entry_points = {
+        'aim': 'aim_plugin',
+        'conjur': 'conjur_plugin',
+        'hashivault_kv': 'hashivault_kv_plugin',
+        'hashivault_ssh': 'hashivault_ssh_plugin',
+        'azure_kv': 'azure_keyvault_plugin',
+        'centrify_vault_kv': 'centrify_plugin',
+        'thycotic_dsv': 'dsv_plugin',
+        'thycotic_tss': 'tss_plugin',
+        'aws_secretsmanager_credential': 'aws_secretmanager_plugin',
+    }
+    for x in expected_entry_points.keys():
+        assert x in entry_points.names
```

Comment:
if you were to avoid subtests, though, the correct check would be a oneliner `assert set(expected_entry_points.keys()) <= set(entry_points.names)`